src/nft/components/collection/CollectionStats.tsx

```diff
@@ -0,0 +1,328 @@
+            <SocialsIcon href={collectionStats.discordUrl}>
+              <DiscordIcon
+                fill={themeVars.colors.darkGray}
+                color={themeVars.colors.darkGray}
+                width="26px"
+                height="26px"
+              />
+            </SocialsIcon>
+          ) : null}
+          {collectionStats.twitter ? (
+            <SocialsIcon href={'https://twitter.com/' + collectionStats.twitter}>
+              <TwitterIcon
+                fill={themeVars.colors.darkGray}
+                color={themeVars.colors.darkGray}
+                width="26px"
+                height="26px"
+              />
+            </SocialsIcon>
+          ) : null}
+
+          {collectionStats.instagram ? (
+            <SocialsIcon href={'https://instagram.com/' + collectionStats.instagram}>
+              <InstagramIcon fill={themeVars.colors.darkGray} width="26px" height="26px" />
+            </SocialsIcon>
+          ) : null}
+          {collectionStats.externalUrl ? (
+            <SocialsIcon href={collectionStats.externalUrl}>
+              <ExternalIcon fill={themeVars.colors.darkGray} width="26px" height="26px" />
+            </SocialsIcon>
+          ) : null}
+        </Row>
+      </Row>
+      {isMobile &&
+        (collectionStats.discordUrl ||
+          collectionStats.twitter ||
+          collectionStats.instagram ||
+          collectionStats.externalUrl) && (
+          <MobileSocialsPopover
+            collectionStats={collectionStats}
+            collectionSocialsIsOpen={collectionSocialsIsOpen}
+            toggleCollectionSocials={toggleCollectionSocials}
+          />
+        )}
+    </Row>
+  )
+}
+
+const CollectionDescription = ({ description }: { description: string }) => {
+  const [showReadMore, setShowReadMore] = useState(false)
+  const [readMore, toggleReadMore] = useReducer((state) => !state, false)
+  const baseRef = useRef<HTMLDivElement>(null)
+  const descriptionRef = useRef<HTMLDivElement>(null)
+
+  useEffect(() => {
+    if (
+      baseRef &&
+      descriptionRef &&
+      baseRef.current &&
+      descriptionRef.current &&
+      (descriptionRef.current.getBoundingClientRect().width >= baseRef.current?.getBoundingClientRect().width - 112 ||
+        descriptionRef.current.getBoundingClientRect().width >= 590)
+    )
+      setShowReadMore(true)
+  }, [descriptionRef, baseRef])
+
+  return (
+    <Box ref={baseRef} marginTop={{ mobile: '12', tabletSm: '16' }} style={{ maxWidth: '680px' }}>
+      <Box
+        ref={descriptionRef}
+        className={clsx(styles.description, styles.nameText, readMore && styles.descriptionOpen)}
+      >
+        <ReactMarkdown
+          source={description}
+          allowedTypes={['link', 'paragraph', 'strong', 'code', 'emphasis', 'text']}
+          renderers={{ paragraph: 'span' }}
+        />
+      </Box>
+      <Box as="span" display={showReadMore ? 'inline' : 'none'} className={styles.readMore} onClick={toggleReadMore}>
```

Comment:
Should the entire description be the click target here?

Reply:
no the spec is for the user to click show more and the description expands